src/modules/auth/components/login/login-form/login-form.sx.ts

```diff
@@ -0,0 +1,37 @@
+export const loginFormStyles = {
+  wrapper: {
+    border: '1px solid rgb(176, 176, 176)',
+    borderRadius: '8px',
+  },
+
+  dividerContainer: {
+    width: '100%',
+    marginBottom: '16px',
+    fontWeight: 500,
+    fontSize: '0.8rem',
+  },
+
+  divider: {
+    display: 'flex',
+    alignItems: 'center',
+    width: '100%',
+
+    '&::before': {
+      content: "''",
+      display: 'block',
+      width: '100%',
+      height: '1px',
+      backgroundColor: '#dddddd',
```

Comment:
make sure that mostly used colors are stored as constants

Reply:
fixed